charts/gardener-extension-shoot-rsyslog-relp/templates/rbac.yaml

```diff
@@ -85,9 +47,7 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - configmaps
   - secrets
-  - events
   verbs:
```

Comment:
Nice cache! You could also check https://github.com/gardener/gardener-extension-registry-cache/pull/85/commits/5e41f55a16917134f03506f770a1b4340cdef310. Some of the rules can be moved from the ClusterRole to a Role.